src/policy/fees.h

```diff
@@ -192,9 +195,11 @@ class CBlockPolicyEstimator
      */
     static constexpr std::chrono::hours MAX_FILE_AGE{60};
 
+    bool m_read_stale_estimates;
```

Comment:
2fec7f362d8540cda5ff7e3e4dfb2cb751364e06

this should be const

Reply:
Thank you